modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -140,6 +140,8 @@ import TableHeader from '@tiptap/extension-table-header';
 import TableRow from '@tiptap/extension-table-row';
 import Placeholder from '@tiptap/extension-placeholder';
 
+import { klona } from 'klona';
```

Comment:
I tried `structuredClone` but it was too strict about not cloning Vue's proxy objects.